src/Server.ts

```diff
@@ -1,5 +1,6 @@
-import { IConfig } from './config/IConfig';
+import { IConfig, notFound, errorHandler } from './index';
```

Comment:
- IConfig should be imported from config folder.
- notFoundRoute and errorHandler should be imported from libs folder.
- change notFound to notFoundRoute